commerce_coordinator/apps/lms/views.py

```diff
@@ -247,22 +273,31 @@ def post(self, request) -> Response:
                             'attribute.', status=HTTP_400_BAD_REQUEST)
 
         if not order_line_item_id:
-            logger.error(f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
-                         f"course_id: {course_id} the enrollment_attributes array requires an orders: line_item_id "
-                         f"attribute.")
-            return Response('the enrollment_attributes array requires an orders: line_item_id '
-                            'attribute.', status=HTTP_400_BAD_REQUEST)
+            log_msg = (
+                f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
+                f"entitlement_id: {entitlement_id}. Line item does not exist."
+            ) if entitlement_refund else (
+                f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
+                f"course_id: {course_id} the enrollment_attributes array requires an orders: line_item_id "
+                f"attribute."
+            )
+            logger.error(log_msg)
+
+            response_msg = 'Line item does not exist.' if entitlement_refund else \
+                'the enrollment_attributes array requires an orders: line_item_id attribute.'
+            return Response(response_msg, status=HTTP_400_BAD_REQUEST)
```

Comment:
NIT
```suggestion
if not order_line_item_id:
    log_msg_prefix = f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
    
    if entitlement_refund:
        log_msg = f"{log_msg_prefix}entitlement_id: {entitlement_id}. Line item does not exist."
        response_msg = "Line item does not exist."
    else:
        log_msg = (
            f"{log_msg_prefix}course_id: {course_id}. The enrollment_attributes array "
            "requires an orders: line_item_id attribute."
        )
        response_msg = "The enrollment_attributes array requires an orders: line_item_id attribute."

    logger.error(log_msg)
    return Response(response_msg, status=HTTP_400_BAD_REQUEST)
```

Reply:
if in future we need to update the whole log together for entitlement, we have to update the prefix. Moreover, inline conditioning is better for such cases